content/working_with/console/widgets/userGroups.md

```diff
@@ -4,24 +4,70 @@ title: User Group Management
 category: Cloudify Console
 draft: false
 ---
-Displays the list of user groups and enables their management. This widget is only available to admin users.
- 
+Displays the list of user groups and enables their management. 
+
+{{% note %}}
+This widget is only available to admin users.
+{{% /note %}}
+
+![User Groups]( /images/ui/widgets/usergroups-mgmt.png )
+
+## Features
+
+### User Groups information
+
 The widget displays the following information regarding each of the user groups:
 
 * **Name**
 * **LDAP group** When working with an ldap-based external authentication system, this fields identifies the LDAP user group which is connected to the current Cloudify user-group. 
 * **Admin** If checked, all users who are members of this groups will have the role of sys-admins on the manager. 
 * **# Users** number of users who are members of the group
 * **# Tenants** number of tenants the user-group is assigned with.
+
  
+### User Groups actions
+
 The hamburger menu on the right of every tenant allows performing the following operations:
 
 * **Adding/removing users to/from the group** available only if managing the users in Cloudify itself
 * **Adding/removing user groups to/from the tenant**
 * **Deleting the user groups** - possible only if there are no users who are members in the groups, and the group is not assigned with any tenants. 
 Also, using the “Add” button on the right top corner of the widget, you will be able to create new user groups. 
 
-![manage-usergroups]( /images/ui/widgets/manage-usergroups.png )
 
-#### Widget Settings 
+#### Adding a User Group
+
+Users groups are not mandatory when you manage users in Cloudify, however creating groups might enable you to manage your users more efficiently. You can create groups of users and assign them to one or more tenants, specifying a tenant-role that will apply to all the users in the group.
+Please notice that by belonging to several groups, users might be assigned to a tenant with more than one role. As each role represents a set of permissions, if even one of those roles allows the users to perform an action in the tenant, they will indeed be able to perform it.
+
+1. Click **Add** in the User Groups Management widget.
+2. Enter a name for the group.   
+   The name must contain at least five alphanumeric characters and begin with a letter. It can also include the following characters `-`, `_`, or `.`.
```

Comment:
```suggestion
   The name must contain at least five alphanumeric characters and begin with a letter. It can also include the following characters: `-`, `_` or `.`.
```